### PR TITLE
Add HipChat documentation

### PR DESCRIPTION
**Pull request by justincampbell**

### CODE REVIEW

File: consul/alerts/notification-methods.html.md [hunk 1]
```diff
@@ -13,11 +17,41 @@ An email can be set and receive alerts trigged by Atlas. Email
...
+## HipChat
+
+Notifications can be sent to a [HipChat](https://hipchat.com) room. There are 2
+ways to configure HipChat notifications:
+
+### User Account
+
+This API token can post to any room that your user account has access to.
+
+1. In your <strong>Account Settings</strong>, click [API
```
Comment (pearkes) [inline consul/alerts/notification-methods.html.md]: I think you can do `**` here, or something

**Comment by pearkes** [comment]
One not on the markup but otherwise LGTM

File: consul/alerts/notification-methods.html.md [hunk 2]
```diff
@@ -3,9 +3,13 @@ title: "Alert Notification Methods"
...
 # Alert Notification Methods
 
-Atlas currently supports three different types of notification methods.
-All three are outlined below. Alert methods can be configured in the
-"Integrations" tab of an environment.
+Atlas currently supports many different types of notification methods, outlined
+below.
+
+Notification methods can be managed by visiting the
+<strong>Configuration</strong> page for your user or organization.
```
Comment (sethvargo) [inline consul/alerts/notification-methods.html.md]: `**`

File: consul/alerts/notification-methods.html.md [hunk 3]
```diff
@@ -3,9 +3,13 @@ title: "Alert Notification Methods"
...
+Notification methods can also be overridden for a specific build configuration,
```
Comment (sethvargo) [inline consul/alerts/notification-methods.html.md]: I think we should link these to their glossary pages

File: consul/alerts/notification-methods.html.md [hunk 4]
```diff
@@ -13,11 +17,41 @@ An email can be set and receive alerts trigged by Atlas. Email
...
 
-To configure Email:
+To configure Email notifications:
 
 1. Enter the email into the "Integrations" tab of the environment in Atlas
 1. Optionally, test the alert
 
+## HipChat
+
+Notifications can be sent to a [HipChat](https://hipchat.com) room. There are 2
```
Comment (sethvargo) [inline consul/alerts/notification-methods.html.md]: "two" - http://www.writersdigest.com/editor-blogs/questions-and-quandaries/formatting/when-do-i-spell-out-numbers

File: consul/alerts/notification-methods.html.md [hunk 5]
```diff
@@ -29,16 +63,19 @@ To configure Slack:
 1. Enter the webhook URL into the field provided in the "Integrations" tab of the environment
 1. Optionally, test the alert
 
-## Pagerduty
+## PagerDuty
```
Comment (sethvargo) [inline consul/alerts/notification-methods.html.md]: Thanks! This was bothering me :smile:

File: terraform/runs/notifications.html.md [hunk 6]
```diff
@@ -4,8 +4,9 @@ title: "Terraform Run Notifications"
 
 # Terraform Run Notifications
 
-Atlas can send run notifications to your organization via Email, Slack, or
-Webhooks. The following events are configurable:
+Atlas can send run notifications to your organization via one of our [supported
```
Comment (sethvargo) [inline terraform/runs/notifications.html.md]: Can we double-check that this multi-line markdown link is parsed correctly?

File: consul/alerts/notification-methods.html.md (hunk 3 above)
Comment (justincampbell) [inline consul/alerts/notification-methods.html.md]: Link what?

File: terraform/runs/notifications.html.md (hunk 6 above)
Comment (justincampbell) [inline terraform/runs/notifications.html.md]: It is

**Pull request merged**